internal/daemon/controller/handlers/targets/target_service.go

```diff
@@ -40,6 +40,7 @@ import (
 	"github.com/hashicorp/boundary/internal/types/subtypes"
 	"github.com/hashicorp/boundary/sdk/pbs/controller/api/resources/scopes"
 	pb "github.com/hashicorp/boundary/sdk/pbs/controller/api/resources/targets"
+	fm "github.com/hashicorp/boundary/version"
```

Comment:
Does this need to be aliased?

Reply:
I wanted to avoid an any confusion between the package name and when we declare the variable name `version` in this file. I used fm for feature manager

Reply:
Sounds good to me